<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string extracomment="que miras?">Calculadora</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QTextEdit" name="output_textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="input_lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="5" colspan="3">
    <layout class="QGridLayout" name="gridLayout_funciones">
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_raiz">
       <property name="text">
        <string>sqrt(</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_neg">
       <property name="text">
        <string>(-)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_grados">
       <property name="text">
        <string>° ' &quot;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_coma">
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_fraccion">
       <property name="text">
        <string>a/b</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_cuadrado">
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_RCL">
       <property name="text">
        <string>RCL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_ln">
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_cos">
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_sin">
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_Mem">
       <property name="text">
        <string>M+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_parentesis_c">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_tan">
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_parentesis_a">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_hyp">
       <property name="text">
        <string>hyp</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_ENG">
       <property name="text">
        <string>ENG</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_inv">
       <property name="text">
        <string>x⁻¹</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_cubo">
       <property name="text">
        <string>x³</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_log">
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_potencia">
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_Pol">
       <property name="text">
        <string>Pol(</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_nCr">
       <property name="text">
        <string>nCr</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="15" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_numeros">
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_igual">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_num2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_num8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_num4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_num1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_num5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_num0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_menos">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_dividir">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_punto">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_num6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_num9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_num3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_exp">
       <property name="text">
        <string>EXP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_AC">
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_num7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_multiplicar">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_mas">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_DEL">
       <property name="text">
        <string>DEL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_ans">
       <property name="text">
        <string>ANS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>input_lineEdit</tabstop>
  <tabstop>pushButton_mas</tabstop>
  <tabstop>pushButton_menos</tabstop>
  <tabstop>pushButton_multiplicar</tabstop>
  <tabstop>pushButton_dividir</tabstop>
  <tabstop>pushButton_igual</tabstop>
  <tabstop>pushButton_AC</tabstop>
  <tabstop>pushButton_DEL</tabstop>
  <tabstop>pushButton_num1</tabstop>
  <tabstop>pushButton_num2</tabstop>
  <tabstop>pushButton_num3</tabstop>
  <tabstop>pushButton_num4</tabstop>
  <tabstop>pushButton_num5</tabstop>
  <tabstop>pushButton_num6</tabstop>
  <tabstop>pushButton_num7</tabstop>
  <tabstop>pushButton_num8</tabstop>
  <tabstop>pushButton_num9</tabstop>
  <tabstop>pushButton_num0</tabstop>
  <tabstop>pushButton_punto</tabstop>
  <tabstop>pushButton_exp</tabstop>
  <tabstop>pushButton_ans</tabstop>
  <tabstop>pushButton_inv</tabstop>
  <tabstop>pushButton_nCr</tabstop>
  <tabstop>pushButton_Pol</tabstop>
  <tabstop>pushButton_cubo</tabstop>
  <tabstop>pushButton_fraccion</tabstop>
  <tabstop>pushButton_raiz</tabstop>
  <tabstop>pushButton_cuadrado</tabstop>
  <tabstop>pushButton_potencia</tabstop>
  <tabstop>pushButton_log</tabstop>
  <tabstop>pushButton_ln</tabstop>
  <tabstop>pushButton_neg</tabstop>
  <tabstop>pushButton_grados</tabstop>
  <tabstop>pushButton_hyp</tabstop>
  <tabstop>pushButton_sin</tabstop>
  <tabstop>pushButton_cos</tabstop>
  <tabstop>pushButton_tan</tabstop>
  <tabstop>pushButton_RCL</tabstop>
  <tabstop>pushButton_ENG</tabstop>
  <tabstop>pushButton_parentesis_a</tabstop>
  <tabstop>pushButton_parentesis_c</tabstop>
  <tabstop>pushButton_coma</tabstop>
  <tabstop>pushButton_Mem</tabstop>
  <tabstop>output_textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
